x-pack/plugins/enterprise_search/public/applications/workplace_search/app_logic.test.ts

```diff
@@ -69,27 +69,39 @@ describe('AppLogic', () => {
   describe('setContext', () => {
     it('sets context', () => {
       AppLogic.actions.setContext(true);
-
-      expect(AppLogic.values.isOrganization).toEqual(true);
+      expect(AppLogic.values).toEqual({
+        ...DEFAULT_VALUES,
+        isOrganization: true,
+      });
     });
   });
 
   describe('setSourceRestriction', () => {
     it('sets property', () => {
-      mount(DEFAULT_INITIAL_APP_DATA);
+      mount({}, DEFAULT_INITIAL_APP_DATA);
```

Comment:
Just curious, why was this needed?

Reply:
not sure, in theory it shouldn't be needed. I will check again reverting these